cryptographic_estimators/MQEstimator/MQAlgorithms/bjorklund.py

```diff
@@ -71,20 +74,41 @@ def lambda_(self):
             sage: E.lambda_()
             3/10
         """
+        
+        time = inf
+        for params in self._valid_choices():
+            tmp_time = self._compute_time_complexity(params)
+            tmp_memory = self._compute_memory_complexity(params)
+            if self.bit_complexities:
+                tmp_memory = self.problem.to_bitcomplexity_memory(tmp_memory)
+
+            tmp_time += self.memory_access_cost(tmp_memory)
+
+            if tmp_time < time and tmp_memory <= self.problem.memory_bound:
+                time, _ = tmp_time, tmp_memory
+                self._current_minimum_for_early_abort = tmp_time #ver si este se puede usar para el early abort en _are_par
+                for i in params:
+                    self._optimal_parameters[i] = params[i]
+            
+            if tmp_time > time:
+                break
+            
+        self._current_minimum_for_early_abort = inf
+
         return self._get_optimal_parameter('lambda_')
```

Comment:
Here you have to consider the case when the complexity time is tildeO. Notice this is the case in the method `_get_optimal_parameter` in the base clase. 
This is why the test in line 208 fails.